src/assets/js/customizer/color/preview.js

```diff
@@ -425,7 +425,6 @@ export class Preview  {
 		subcolor = paletteSelector.getColor( subcolor, true );
 
 		let alpha    = parent.net.brehaut.Color( color );
-		let subalpha = parent.net.brehaut.Color( subcolor );
 		let css = '';
```

Comment:
Wasnt able to find this in the WP coding standards, but do = signs need to align for JS?